inventory-provisioning-web/src/main/scala/com/normation/inventory/provisioning/endpoint/FusionReportEndpoint.scala

```diff
@@ -84,6 +95,28 @@ class FusionReportEndpoint(
   )
   def checkStatus() = new ResponseEntity("OK", HttpStatus.OK)
 
+
+  /**
+   * Info on current number of elements in queue
+   */
+  @RequestMapping(
+    value = Array("/api/info"),
+    method = Array(RequestMethod.GET)
+  )
+  def queueInfo() = {
+    (ReportProcessor !? GetQueueInfo) match {
+      case QueueInfo(max, current) =>
+        val saturated = (current+1) >= max
+        val code = if(saturated) HttpStatus.TOO_MANY_REQUESTS else HttpStatus.OK
+        val json = s"""{  "queueMaxSize": "$max",  "queueFillCount": "$current",  "queueSaturated":"$saturated"}"""
```

Comment:
Good question. They were string in the example in the issue, but there is no reason. 
Numbers will be better (and boolean for queueSaturated)